.env.example

```
@@ -5,7 +5,7 @@ NATS_PASSWORD= # NATS password (optional)
 NATS_TLS_ENABLED=false # Enable TLS for NATS connection (default: false)
 
 ### HadesAPI Options ###
-API_PORT=8080 # Port for the API to listen on (default: 8080)
+HADESAPI_API_PORT=8080 # Port for the API to listen on (default: 8080)
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)